tr.yml

```diff
@@ -0,0 +1,279 @@
+    loadingFailed: "Yükleme başarısız, lütfen internet bağlantınızı kontrol edin"
+    loadingFailedTitle: "Yükleme başarısız"
+    loadingFailedSubtitle: "Lütfen internet bağlantınızı kontrol edin ve tekrar deneyin"
+    tryAgain: "Tekrar dene"
+    updating: "Güncelleniyor..."
+    installing: "Yükleniyor..."
+    compiling: "Derleniyor..."
+  appHeader:
+    home: "Ana Sayfa"
+    explore: "Keşfet"
+    settings: "Ayarlar"
+    about: "Hakkında"
+  mod:
+    updateAvailable: "Güncelleme mevcut"
+    installed: "Yüklendi"
+    details: "Ayrıntılar"
+    remove: "Kaldır"
+    removeConfirm: "Bu modu kaldırmak istediğinizden emin misiniz??"
+    removeConfirmOk: "Modu kaldır"
+    removeConfirmCancel: "İptal"
+    notCompiled: "Modun derlenmesi gerekiyor"
+    editedLocally: "Mod yerel olarak düzenlendi"
+    noDescription: "(açıklama yok)"
+    users_one: "{{formattedCount}} kullanıcı"
+    users_other: "{{formattedCount}} kullanıcı"
+  modDetails:
+    header:
+      installedVersion: "Yüklü sürüm"
+      latestVersion: "En son sürüm"
+      loading: "Yükleniyor..."
+      loadingFailed: "yükleme başarısız"
+      modId: "mod tanımlayıcı"
+      modVersion: "Mod sürümü"
+      modAuthor:
+        title: "Mod yapımcısı"
+        homepage: "Ana Sayfa"
+        github: "GitHub"
+        twitter: "Twitter"
+      processes:
+        all: "Tüm işlemler"
+        allBut: "All but {{list}}"
+        except: "{{included}} except {{excluded}}"
+        tooltip:
+          target: "Hedef işlem"
+          targets: "Hedef işlemler:"
+          excluded: "Hariç::"
+      update: "Güncelle"
+      updateNotNeeded: "Yüklü sürüm, en son sürümle aynı"
+      install: "Yükle"
+      compile: "Derle"
+      disable: "Devre dışı bırak"
+      enable: "Etkinleştir"
+      edit: "Düzenle"
+      fork: "Fork"
+      remove: "Kaldır"
+    details:
+      title: "Ayrıntılar"
+      noData: "Mod ayrıntıları eksik."
+    settings:
+      title: "Ayarlar"
+      noData: "Kullanılabilir mod ayarı yok."
+      saveButton: "Ayarları kaydet"
+      sampleValue: "Örnek değer"
+      arrayItemAdd: "Yeni öğe ekle"
+      arrayItemRemove: "Öğeyi kaldır"
+    code:
+      title: "Kaynak Kodu"
+      noData: "Mod kaynağı eksik."
+      collapseExtra: "Beni Oku ve Ayarları Daralt"
+    changelog:
+      title: "Değişiklikler"
+      loadingFailed: "Değişiklikler yüklenemedi. GitHub'da görüntülemek için <0>burayı tıklayın</0>."
+    advanced:
+      title: "Gelişmiş"
+      debugLogging:
+        title: "Hata ayıklama günlüğü"
+        description: "Modla ilgili sorunları gidermek için yararlı olabilir."
+        none: "Hiçbiri"
+        modLogs: "Mod günlükleri"
+        detailedLogs: "Ayrıntılı hata ayıklama günlükleri"
+        showLogButton: "Günlük çıktısını göster"
+      modSettings:
+        title: "Mod ayarları"
+        description: "Mod ayarlarınızı kolayca dışa aktarın veya paylaşın."
+        loadButton: "Yükle"
+        loadFormattedButton: "Load formatted"
```

Comment:
Untranslated.

Reply:
Thank you very much for seeing and warning my mistakes. I left some for testing and some I translated incorrectly :). Now only the following string is missing: TRAY_EMERGENCY_EXIT: "Signal &service exit. I wonder what that means? Thank you very much. Have a nice day.

Reply:
"Signal service exit" appears when you hold Shift and right click on the tray icon. It allows to exit Windhawk without admin rights. Perhaps I just need to remove it from the translation, it's a hidden option and it's not that important to have it translated.